tests/src/Search/Workflow/GetSerializer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace tests\eLife\Search\Workflow;
+
+use eLife\ApiClient\ApiClient\SubjectsClient;
+use eLife\ApiSdk\Client\Subjects;
+use eLife\ApiSdk\Serializer\Block;
+use eLife\ApiSdk\Serializer\BlogArticleNormalizer;
+use eLife\ApiSdk\Serializer\ImageNormalizer;
+use eLife\ApiSdk\Serializer\MediumArticleNormalizer;
+use eLife\ApiSdk\Serializer\SubjectNormalizer;
+use LogicException;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Serializer;
+use tests\eLife\Search\HttpClient;
+
+trait GetSerializer
+{
+    private $serializer;
+
+    public function getSerializer()
+    {
+        if (!method_exists($this, 'getHttpClient')) {
+            throw new LogicException('You need to include the HttpClient to initialize the serializer');
+        }
```

Comment:
I had that initially but then when I was creating the Mock trait here: 
https://github.com/elifesciences/search/pull/13/files/20e8dc5dd227b86889fc3e2c968b6c0ad3146efe#diff-92d16925d99a8a21d6f7bb0638029052R42

I realised I had a double-dependency, what would you suggest?